src/features/forceUpdate/ForceUpdate.tsx

```diff
@@ -42,7 +45,26 @@ const buttonText = Platform.select({
   web: t`Actualiser la page`,
 })
 
-export const ForceUpdate = () => {
+type Props = {
+  resetErrorBoundary: () => void
+}
+
+export const ForceUpdate = ({ resetErrorBoundary }: Props) => {
+  const minimalBuildNumber = useMinimalBuildNumber()
+
+  // This first hook will be like a componentWillUnmount
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  useEffect(() => resetErrorBoundary, [])
+
+  // This one is for when feature flip switch back to off later
```

Comment:
Peut être adapter ce commentaire à la valeur de firestore ? Genre "this one is for when minimalBuildNumber gets back to an older value"